src/main/java/org/jitsi/jigasi/util/Util.java

```diff
@@ -262,7 +262,7 @@ public static String stringToMD5hash(String toHash)
      * @param indent the indent to apply to the XML
      * @return the formatted String
      */
-    public static String prettyFormat(String input, int indent)
+    public static String prettyFormatXMLString(String input, int indent)
```

Comment:
Why do we have an unused method here?

Reply:
I've used it for debugging the presence objects stored in the ChatRoomMember. But is now unused as it was only used for a logging message in the deleted `ChatRoomMemberPresenceReader`